packages/at_client/lib/src/encryption_service/abstract_atkey_encryption.dart

```diff
@@ -29,104 +27,76 @@ abstract class AbstractAtKeyEncryption implements AtKeyEncryption {
         'AbstractAtKeyEncryption (${_atClient.getCurrentAtSign()})');
   }
 
-  @visibleForTesting
```

Comment:
This was being used to try to ensure something had been sync'd. Removing as now obsolete.